Sources/FluidMenuBarExtra/FluidMenuBarExtraWindow.swift

```diff
@@ -36,7 +36,9 @@ final class FluidMenuBarExtraWindow<Content: View>: NSPanel {
     private lazy var hostingView: NSHostingView<some View> = {
         let view = NSHostingView(rootView: rootView)
         // Disable NSHostingView's default automatic sizing behavior.
-        view.sizingOptions = []
+        if #available(macOS 13.0, *) {
+            view.sizingOptions = []
+        }
```

Comment:
What are the implications of these changes? Any changes to behavior?

Reply:
As far as I can tell, no. I tested on the macOS 12 VM